Http.cpp

```diff
@@ -194,6 +196,47 @@ unsigned int HTTP::readVoltage(){
   return atoi(voltage);
 }
 
+void HTTP::batteryState(char *voltage){
+  char buffer[64];
+  cleanBuffer(buffer, sizeof(buffer));
+  cleanBuffer(voltage, sizeof(voltage));
+
+  sendCmd(READ_VOLTAGE);
+
+  if (readBuffer(buffer, sizeof(buffer)) == TRUE){
```

Comment:
I think we could create a private method with the code inside the `if` as is duplicated [here](https://github.com/carrascoacd/ArduinoSIM800L/blob/master/Http.cpp#L175)

Reply:
To be honest I'm really an entry level programmer in C++ or in programming in general. Rather than asking for someone to create functions that I would need in my project I did my best to do that myself. Obviously my code is far from perfect and If you improve it it would be awesome.